src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip

```diff
@@ -2,46 +2,40 @@
 @extends "nodejs/init_code.snip"
 
 @snippet generate(sampleFile)
-   //// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "{@sampleFile.className}" ]
-   //// STUB standalone sample "{@sampleFile.className}" /////
-
-   // FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
-
   @let apiMethod = sampleFile.libraryMethod
-     @let sample = apiMethod.samples.get(0)
-       // [START {@sample.regionTag}]
-
-       // FIXME: Insert here boilerplate code not directly related to the method call itself.
-
-       //      calling form: "{@sample.callingForm.toString}"
-       //        region tag: "{@sample.regionTag}"
-       //         className: "{@sampleFile.className}"
-       //          valueSet: "{@sample.valueSet.id}" ("{@sample.valueSet.title}")
-       //       description: "{@sample.valueSet.description}"
-       //        {@sample.valueSet.parameters}
-       //      apiMethod "{@apiMethod.name}" of type "{@apiMethod.type}"
-
-       // [START {@sample.regionTag}_core]
-
-       // FIXME: Insert here code to prepare the request fields, make the call, process the response.
-
-       /*
-       {@standaloneSample(apiMethod, sample)}
-       */
-       // [END {@sample.regionTag}_core]
+    @let sample = apiMethod.samples.get(0)
+      // DO NOT EDIT! This is a generated sample ("{@sample.callingForm}",  "{@sample.valueSet.id}")
+      'use strict';
+      // [START {@sample.regionTag}]
```

Comment:
I'd suggest some more vertical whitespace: a blank line above and below the `START` set and above and below the `END` set.